Core/Graphics/Source/DeviceImpl.cpp

```diff
@@ -93,6 +93,15 @@ namespace Babylon::Graphics
         }
     }
 
+    void DeviceImpl::SetAlphaPremultiplied(bool enabled)
+    {
+        std::scoped_lock lock{m_state.Mutex};
+        m_state.Bgfx.Dirty = true;
+        auto& init = m_state.Bgfx.InitState;
+        init.resolution.reset &= ~BGFX_RESET_TRANSPARENT_BACKBUFFER;
+        init.resolution.reset |= enabled ? BGFX_RESET_TRANSPARENT_BACKBUFFER : 0;
```

Comment:
nit: i find this to be easier to read
```suggestion
        if (enabled)
        {
            init.resolution.reset |= BGFX_RESET_TRANSPARENT_BACKBUFFER;
        }
        else
        {        
            init.resolution.reset &= ~BGFX_RESET_TRANSPARENT_BACKBUFFER;
        }
```

Reply:
I like compactness. Like https://github.com/BabylonJS/BabylonNative/blob/4c270c2af5b1b47ac7f7a4584f2edbcab83388f2/Plugins/NativeEngine/Source/NativeEngine.cpp#L800

Reply:
It's a nit, so no worries either way.